examples/kitty/kitty.mk

```diff
@@ -129,39 +131,6 @@ micropython.elf: mpy-cross libsddf_util_debug.a libco.a
 			ENABLE_FRAMEBUFFER=1 \
 			V=1
 
-musllibc/lib/libc.a:
-	make -C $(MUSL) \
-		C_COMPILER=aarch64-none-elf-gcc \
-		TOOLPREFIX=aarch64-none-elf- \
-		CONFIG_ARCH_AARCH64=y \
-		STAGE_DIR=$(abspath $(BUILD_DIR)/musllibc) \
-		SOURCE_DIR=.
-
```

Comment:
Suggest keeping this here for now.  musl is logically separate and may be useful for other components